src/ChangeHighlight.tsx

```diff
@@ -25,6 +25,7 @@ const ChangeHighlight: React.FC<Props> = ({ children, hideAfter }) => {
       {shadowDOM.map((child, index) => (
         <ShadowChild key={index} child={child} hideAfter={hideAfter} />
       ))}
+      //TODO: Replace it with childrenWithRef after manipulation.
```

Comment:
I'm thinking about a similar thing but this now is kind of `shadowDOM` I created, I don't like it much but if you want to try doing that, create a PR and let's see

Reply:
I don't see now any other way to do such logic.
About SHADOWDOM, It is good enough for now but still have some cases need to be handled 👌👌